.github/workflows/stale.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Automatically close stale issues'
+
+on:
+  schedule:
+    # Runs every day at 8:00 AM CET
+    - cron: '0 7 * * *'
+  workflow_dispatch:
+
+permissions:
+    issues: write
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/stale@v9
```

Comment:
minor: it is better to pin it to the particular version, better for the security https://github.com/DataDog/dd-sdk-android/blob/0606a315338a2026db105ce26e16c6348153dfdc/.github/workflows/stale.yml#L16